distributed/client.py

```diff
@@ -2342,24 +2347,36 @@ def list_datasets(self, **kwargs):
         """
         return self.sync(self.scheduler.publish_list, **kwargs)
 
-    async def _get_dataset(self, name):
+    async def _get_dataset(self, name, default=NO_DEFAULT_PLACEHOLDER):
```

Comment:
@mrocklin , is there a utility variable or other standard way to do this?

Otherwise, happy with the PR.

Reply:
I did a quick grep through the `distributed` repo and have found

```python
_empty_namedtuple = collections.namedtuple("_empty_namedtuple", ())
```
in `metrics.py` - but I do not know if this is the standard. Happy to follow one!

Reply:
I am personally more familiar with using `__no_default__` over a named tuple.  I think that what is here is fine.